pkgs/development/interpreters/python/mk-python-derivation.nix

```
@@ -193,10 +199,27 @@ let
     "setuptools" "wheel"
   ];
 
+  passthru =
+    attrs.passthru or { }
+    // {
+      updateScript = let
+        filename = builtins.head (lib.splitString ":" self.meta.position);
+      in attrs.passthru.updateScript or [ update-python-libraries filename ];
+    }
+    // lib.optionalAttrs (dependencies != []) {
+      inherit dependencies;
+    }
+    // lib.optionalAttrs (optional-dependencies != {}) {
+      inherit optional-dependencies;
+    } // lib.optionalAttrs (build-system != []) {
+      inherit build-system;
+    };
```

```suggestion
    }
    // lib.optionalAttrs (build-system != []) {
      inherit build-system;
    };
```

Keep the same style as before.